test/unit/ngsiv2/examples/subscriptionRequests/bidirectionalSubscriptionRequest.json

```diff
@@ -0,0 +1,26 @@
+{
+  "subject":{
+    "entities": [
+      {
+        "id": "TheFirstLight",
+        "type": "TheLightType"
+      }
+    ],
+    "condition":{
+      "attrs":[
+        "location"
+      ]
+    }
+  },
+  "notification":{
+    "http":{
+      "url":"http://smartGondor.com/notify"
+    },
+    "attrs":[
+      "location"
+    ],
+    "attrsFormat": "normalized"
+  },
+  "expires": "P1M",
```

Comment:
This should be ISO8601, e.g:

```
"expires": "2016-04-05T14:00:00.00Z"
```

The fix should include also the tests using this .json as if this error has passed unoticed, the tests are somehow "weak".

Reply:
The problem was that I missed that part of the NGSIv2 specification and this JSON file which is used to intercept the requests to the CB was wrong. This is the problem of unit tests with mocks: if mocks are wrong, tests will be passed but functionality will not be correct. We would have detected this problem at a later stage with the integration tests. Anyway, thanks for having detected this bug!!! :)

Reply:
Fixed in 6b52422fca4124bf19334d7d1e68aceb0768d5e8

Reply:
(Semi-offtopic)

> This is the problem of unit tests with mocks: if mocks are wrong, tests will be passed but functionality will not be correct

Fully agree. This is why I insist so much in the importance of doing e2e integration testing with real components ;)